src/__internal__/sticky-footer/sticky-footer.component.js

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import throttle from "lodash/throttle";
+
+import StyledStickyFooter from "./sticky-footer.style";
+
+const SCROLL_THROTTLE = 50;
+
+const StickyFooter = ({ children, containerRef, disableSticky, ...rest }) => {
+  const [isSticky, setIsSticky] = useState(true);
+  const footerRef = useRef();
+
+  const checkFooterPosition = useCallback(
+    throttle(() => {
+      let fullyScrolled;
+      const content = containerRef.current;
+      const stickyOffset = footerRef.current.clientHeight / 2;
+
+      if (
+        content.scrollHeight - content.scrollTop - stickyOffset <=
+        content.clientHeight
+      ) {
+        fullyScrolled = true;
```

Comment:
You could update this so it doesn't need any of the `if` statements and `setIsSticky` to the result of the condition.
```suggestion
     const fullyScrolled = content.scrollHeight - content.scrollTop - stickyOffset <= content.clientHeight;
     
     setIsSticky(fullyScrolled);
```
Not a big deal so if you prefer it the way you have I'm happy to leave it and resolve this conversation

Reply:
Yeah good spot, I laid it out like that because I had some other code in originally, but didn't simplify this when I removed the other stuff - I'll change it